### ISSUE TITLE
Guide breakage

### ISSUE THREAD

**Issue by MaiHameed** [OpenLiberty/guide-docker#72]
Docker Intro
---
* End-to-end on Mac
* Running Java `openj9-0.14.0, JRE 11`

Running `mvn install` automatically triggers the tests, which obviously fail since no containers are running:
```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running it.io.openliberty.guides.rest.EndpointTest
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.apache.cxf.common.util.ReflectionUtil$11 (file:/Users/maihameed%40ibm.com/.m2/repository/org/apache/cxf/cxf-core/3.2.6/cxf-core-3.2.6.jar) to field java.net.Authenticator.theAuthenticator
WARNING: Please consider reporting this to the maintainers of org.apache.cxf.common.util.ReflectionUtil$11
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.519 sec <<< FAILURE! - in it.io.openliberty.guides.rest.EndpointTest
testGetProperties(it.io.openliberty.guides.rest.EndpointTest)  Time elapsed: 1.519 sec  <<< FAILURE!
java.lang.AssertionError: Incorrect response code from http://localhost:9080/LibertyProject/ expected:<200> but was:<404>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at it.io.openliberty.guides.rest.EndpointTest.testGetProperties(EndpointTest.java:53)


Results :

Failed tests:
  EndpointTest.testGetProperties:53 Incorrect response code from http://localhost:9080/LibertyProject/ expected:<200> but was:<404>

Tests run: 1, Failures: 1, Errors: 0, Skipped: 0
```

The Test section of the guide makes no mention of the `EndpointTest` nor does it prompt the user to run any sort of tests. 

![image](https://user-images.githubusercontent.com/26514093/74042248-29e43d80-4995-11ea-8c19-7ed6560af6c9.png)

In addition, telling the user to update the file to a new endpoint won't come into effect unless the Docker image is rebuilt, the instructions do not reflect this, and instead tells the user to simply run `mvn package` and view the updated endpoint which does not work.

Guide will probably need to be looked into.

This will probably be addressed in the `dev-mode` update.

_Originally posted by @MaiHameed in https://github.com/OpenLiberty/guides-common/issues/390#issuecomment-583456209_

**Comment by gkwan-ibm** [comment]
will be addressed by #74

**Comment by gkwan-ibm** [comment]
close
- #74 was closed
